### ISSUE TITLE
UUID Compatibility not working?

### ISSUE THREAD

**Issue by acidbyt3** [Xephi/AuthMeReloaded#985]
Is UUID compatibility implemented?

I'm asking because I didn't find any UUID being saved on the MySQL database and also because an old user came back with a new name and they had to re-register.
On the database, the user ended up listed twice - one with the old, and another with the new name.

**Comment by sgdc3** [comment]
authme is an alternative to uuid/mojang authentication, atm we don't have any " hybrid mode " that allows you to register by your uuui, I'm sorry.

**Comment by acidbyt3** [comment]
In that case you may want to update your plugin's description on Spigot, as it states:
**Full compatible with UUIDs** and Craftbukkit/Spigot 1.8.X!
Use it to secure your Offline mode server **or to incrase your Online mode server's protection!**

**Comment by ljacqu** [comment]
You're right – 1.8 is an old version and we support even 1.10.X :wink:

Jokes aside, if we offered uuid compatibility how would you expect it to work? Go with the uuid if exists and otherwise keep the name? This may make sense in an online server only.

**Comment by acidbyt3** [comment]
Basically assign the password to a UUID if in online mode, instead of a username.
I realise now this is probably not what you are trying to achieve at all, that the plugin is only intended to work with names. I just wrongly assumed that UUID compatibility was indeed integrated as the description suggests.

You are correct, it is for an online server only, however since we run a bungee network only bungee can be in online mode. All the other servers are required to be in offline mode, which is what I was concerned about.


Here is the perfect example as to why I would like this feature:
The other day someone managed to get into one of our offline-mode servers (connected to our online-mode bungee). IP forward is set and everything but they still managed to get in.

We use WorldGuard HostKeys so they were unable to join under an admin's name **however** lets say the admin name was Player1, the offender logged in as Player1_, and their UUID was being reported as the same as the actual admin (Player1). Therefore they had access to all the permissions Player1 had.

I took all the precautions I could think of, even as far as using HostKeys (which they didn't get past) but same UUID under a different name? I never saw that coming... though I may be missing something obvious here (don't shoot if I'm having a noob moment haha)


The point being - that was exactly what I was attempting to prevent. This is why I was hoping to find a plugin that would allow a login password per UUID and your plugin seemed to be the one, only I found out the hard way it didn't. haha

I hope it makes a bit more sense now as to why I was hoping for this feature.
Peace :)
